<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mer::Internal::MerEmulatorDeviceWidget</class>
 <widget class="QWidget" name="Mer::Internal::MerEmulatorDeviceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>508</width>
        <height>240</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QWidget" name="widget" native="true">
         <layout class="QFormLayout" name="formLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::ExpandingFieldsGrow</enum>
          </property>
          <item row="3" column="0">
           <widget class="QLabel" name="emulatorLabel">
            <property name="text">
             <string>Virtual machine:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="emulatorVmLabelEdit">
            <property name="text">
             <string/>
            </property>
            <property name="textInteractionFlags">
             <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="configFolderLabel">
            <property name="text">
             <string>&quot;config&quot; folder:</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLabel" name="configFolderLabelEdit">
            <property name="text">
             <string/>
            </property>
            <property name="textInteractionFlags">
             <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="sshFolderLabel">
            <property name="text">
             <string>&quot;ssh&quot; folder:</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QLabel" name="sshFolderLabelEdit">
            <property name="text">
             <string/>
            </property>
            <property name="textInteractionFlags">
             <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
            </property>
           </widget>
          </item>
          <item row="10" column="0">
           <widget class="QLabel" name="userNameLabel">
            <property name="text">
             <string>Username:</string>
            </property>
            <property name="buddy">
             <cstring>userLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="10" column="1">
           <widget class="QLineEdit" name="userLineEdit">
            <property name="text">
             <string notr="true"/>
            </property>
           </widget>
          </item>
          <item row="11" column="0">
           <widget class="QLabel" name="sshKeyLabel">
            <property name="text">
             <string>Ssh key:</string>
            </property>
           </widget>
          </item>
          <item row="11" column="1">
           <widget class="QLabel" name="sshKeyLabelEdit">
            <property name="text">
             <string>none</string>
            </property>
            <property name="textInteractionFlags">
             <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
            </property>
           </widget>
          </item>
          <item row="13" column="0">
           <widget class="QLabel" name="connectionTimeoutLabel">
            <property name="text">
             <string>Timeout:</string>
            </property>
            <property name="buddy">
             <cstring>timeoutSpinBox</cstring>
            </property>
           </widget>
          </item>
          <item row="13" column="1">
           <widget class="QSpinBox" name="timeoutSpinBox">
            <property name="suffix">
             <string>s</string>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="singleStep">
             <number>10</number>
            </property>
            <property name="value">
             <number>1000</number>
            </property>
           </widget>
          </item>
          <item row="14" column="0">
           <widget class="QLabel" name="freePortsLabel">
            <property name="text">
             <string>Free ports:</string>
            </property>
            <property name="buddy">
             <cstring>portsLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="14" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLineEdit" name="portsLineEdit">
              <property name="toolTip">
               <string>You can enter lists and ranges like this: 1024,1026-1028,1030</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="portsWarningLabel">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="sshPortLabel">
            <property name="text">
             <string>Ssh port:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLabel" name="sshPortLabelEdit">
            <property name="text">
             <string>none</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="hostLabel">
            <property name="text">
             <string>Host:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="hostLabelEdit">
            <property name="text">
             <string>localhost</string>
            </property>
           </widget>
          </item>
         </layout>
         <zorder>userLineEdit</zorder>
         <zorder>userNameLabel</zorder>
         <zorder>connectionTimeoutLabel</zorder>
         <zorder>timeoutSpinBox</zorder>
         <zorder>freePortsLabel</zorder>
         <zorder>emulatorVmLabelEdit</zorder>
         <zorder>emulatorLabel</zorder>
         <zorder>sshKeyLabel</zorder>
         <zorder>sshKeyLabelEdit</zorder>
         <zorder>configFolderLabel</zorder>
         <zorder>sshFolderLabel</zorder>
         <zorder>configFolderLabelEdit</zorder>
         <zorder>sshFolderLabelEdit</zorder>
         <zorder>sshPortLabel</zorder>
         <zorder>sshPortLabelEdit</zorder>
         <zorder>hostLabel</zorder>
         <zorder>hostLabelEdit</zorder>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
